data-modeling/delegate-types/README.md

```diff
@@ -0,0 +1,64 @@
+# Data modeling: Delegate types
+
+This example shows you how to model [delegate types](https://edgeapi.rubyonrails.org/classes/ActiveRecord/DelegatedType.html) using Prisma, and use [Prisma Client](https://www.prisma.io/client) in a **Node.js script** to read and write data.
+
+This example consists of a `script.ts` file that contains example Prisma Client queries for delegate types
+
+
+## What are Delegate types?
+
+TODO?
```

Comment:
TODO

Reply:
I was conflicted on whether I should add an explanation on what delegate/ union types are in this section, or we could post something on dev.to